Formula/dotnet.rb

```diff
@@ -22,10 +17,10 @@ class Dotnet < Formula
   depends_on "pkg-config" => :build
   depends_on xcode: :build
   depends_on arch: :x86_64
```

Comment:
If I remember, .NET 6 should have Apple Silicon (ARM) support. If so, we should remove arch restriction.
```suggestion
```